core/api/common_api.hpp

```diff
@@ -43,9 +66,66 @@ namespace fc::api {
   template <typename A, typename F>
   void visitCommon(A &&a, const F &f) {
     f(a.AuthNew);
+    f(a.AuthVerify);
     f(a.NetAddrsListen);
     f(a.NetConnect);
     f(a.NetPeers);
     f(a.Version);
   }
+
+  inline void makeCommonApi(
```

Comment:
`CommonApi` declaration is used in many places, please move this method (and jwt include) to other header.  

maybe rename to makeAuthApi or fillAuthApi